internal/oci/uvm.go

```diff
@@ -315,6 +320,9 @@ func SpecToUVMCreateOpts(ctx context.Context, s *specs.Spec, id, owner string) (
 	if IsLCOW(s) {
 		lopts := uvm.NewDefaultOptionsLCOW(id, owner)
 		lopts.MemorySizeInMB = ParseAnnotationsMemory(ctx, s, annotationMemorySizeInMB, lopts.MemorySizeInMB)
+		lopts.LowMmioGapInMB = parseAnnotationsUint64(ctx, s.Annotations, annotationMemoryLowMmioGapInMB, lopts.LowMmioGapInMB)
+		lopts.HighMmioBaseInMB = parseAnnotationsUint64(ctx, s.Annotations, annotationMemoryHighMmioGapInMB, lopts.HighMmioBaseInMB)
```

Comment:
It looks like you have the annotations for `HighMmioGap` and `HighMmioBase` flipped here.